app/src/main/java/app/musikus/usecase/activesession/PauseUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package app.musikus.usecase.activesession
+
+import app.musikus.utils.TimeProvider
+import kotlinx.coroutines.flow.first
+import kotlin.time.Duration.Companion.seconds
+
+class PauseUseCase(
```

Comment:
Well, the idea was that all functions are understood in the context of the current usecase group. So when we are in the `usecase/activesession` folder, it is clear that `PauseUseCase` pauses the active session.

But, yes since kotlin kind of does not really handle multiple same named classes that good we might reame...